spec/services/create_repair_spec.rb

```diff
@@ -85,43 +88,6 @@
     end
   end
 
-  it 'posts a default description, if none was provided' do
```

Comment:
Any reason why this was removed?

Reply:
Because it's no longer relevant compared to when it was first added – the relevant commit message has more detail.